modules/govuk_postgresql/templates/usr/local/bin/autopostgresqlbackup.erb

```diff
@@ -637,9 +637,8 @@ if [ -s "$LOGERR" ]
     STATUS=0
 fi
 
-# Clean up Logfile
-rm -f "$LOGFILE"
-rm -f "$LOGERR"
+# Clean up Logfiles older than two weeks
+/usr/bin/find /var/lib/autopostgresqlbackup -iname *.log -mtime +14 -type f -delete
```

Comment:
If we're setting the $BACKDIR variable earlier in the file I think it'd better to set it here because if we change location of backups then they won't get removed due to the hardcoded location.